README.md

```diff
@@ -58,6 +58,18 @@ git clone git@github.com:nuodb/nuodb-collector.git
 cd nuodb-collector
 ```
 
+Create a .env file in root folder and add
```

Comment:
This implies that the user has InfluxDB knowledge and understands these variables. 
I would suggest leaving clear instructions/examples on how to configure them. It could be a script that the user can source before starting docker-compose. This will also help in removing the hard-codded variables in the CircleCI config yaml.